### PR TITLE
Frontend server settings

### PR DESCRIPTION
**Pull request by SchrodingersGat**
Adds more comprehensive settings method for setting frontend settings

### CODE REVIEW

File: InvenTree/web/templatetags/spa_helper.py [hunk 1]
```diff
@@ -7,14 +8,10 @@
 from django.conf import settings
 from django.utils.safestring import mark_safe
 
-logger = getLogger("InvenTree")
+logger = getLogger("inventree")
```
Comment (matmair) [inline InvenTree/web/templatetags/spa_helper.py:11]: Comment: This is breaking for ppl that use custom logging drivers

**Review by matmair — COMMENTED** [review]
I think using „frontend“ has a potential to confuse users wether the old or new ui is meant. I get a feeling that the old UI will stick around for a bit so i think there should be a good distinction.

**Comment by SchrodingersGat** [comment]
@matmair any comment on the currently failing `Deepsource:Python` issue (`mark_safe`)?

**Comment by matmair** [comment]
Mark_safe is generally a security risk, I will review these specific instances

**Pull request merged**